notebooks/265-qrcode-monster/265-qrcode-monster.ipynb

```diff
@@ -0,0 +1,974 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        seed: int = 42,
</pre><p>This way you have to use a seed every time. <code>seed: Optional[int] = 42</code>  and <code>if seed is not None: np.random.seed(int(seed))</code> makes it more flexible.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
makes sense! fixed